### ISSUE TITLE
get a very low precision

### ISSUE THREAD

**Issue by hajungong007** [dkorobchenko-nv/tensorrt-demo#1]
hi, I get a very low precision using tensorRT, but get 0.9580 using tf. here is my code:

def network(inputs, is_training=True, **kwargs):
    #Lenet + BatchNorm
    with tf.variable_scope('net'):
        with slim.arg_scope([slim.conv2d, slim.fully_connected],
                            activation_fn=tf.nn.relu,
                            weights_initializer=tf.truncated_normal_initializer(0.0, 0.01),
                            weights_regularizer=slim.l2_regularizer(0.0005)):
            with slim.arg_scope([slim.conv2d],
                                normalizer_fn=slim.batch_norm,
                                normalizer_params={'is_training': is_training}):
                net = slim.conv2d(inputs, 32, [5, 5], scope='conv1')
                net = slim.max_pool2d(net, [2, 2], scope='pool1')
                net = slim.conv2d(net, 64, [5, 5], scope='conv2')
                net = slim.max_pool2d(net, [2, 2], scope='pool2')
                net = slim.flatten(net)
                net = slim.fully_connected(net, 1024, scope='fc1')
                net = slim.dropout(net, 0.5, scope='dropout', is_training=is_training)
                net = slim.fully_connected(net, 10, activation_fn=None, scope='fc2')
        return net

**Comment by dkorobchenko** [comment]
Hi, the issue could occur due to fuzzy support of some layers, depending on version of TF and TRT.
1) Please try to use newer code (recently updated), there are newer versions of TF (1.10) and TRT (5.0). I recommend to use docker containers (included in the repo) to reproduce the result.
2) Another option is to try substituting slim.flatten() with tf.reshape. I noticed, that sometimes it interprets CHW/HWC improperly. Check updated model.py
